src/main/res/values/strings.xml

```diff
@@ -1303,17 +1303,17 @@
 	</plurals>
 
 	<!-- 2020-09-19 -->
-	<string name="action_save_image_success_no_path">Image saved successfully</string>
+	<string name="action_save_image_success_no_path">Image saved</string>
 
 	<!-- 2020-10-20 -->
 	<string name="pref_behaviour_imagevideo_tap_close_key" translatable="false">pref_behaviour_imagevideo_tap_close</string>
 	<string name="pref_behaviour_imagevideo_tap_close_title">Close images/videos on tap</string>
-	<string name="pref_behaviour_imagevideo_tap_close_summary">Does not affect videos if playback controls are enabled</string>
+	<string name="pref_behaviour_imagevideo_tap_close_summary">Does not affect videos if playback controls are on</string>
```

Comment:
```suggestion
	<string name="pref_behaviour_imagevideo_tap_close_summary">Only affects videos without playback controls.</string>
```

Reply:
This doesn't preserve the meaning, it's specifically referring to the video playback controls preference, which can be enabled or disabled.